### PR TITLE
add monitoring tab to admin_ui page

### PR DESCRIPTION
**Pull request by m-kharbat**

### CODE REVIEW

File: pages/docs/connect/admin_ui.html [hunk 1]
```diff
@@ -10,29 +10,24 @@
...
-- Latest news from Crate.
+- Settings Menu Which allows to update the language settings and provides a link to Crate documentation and the latest news from Crate.
 
 The overview tab displays similar information to the bar but with more details on data availability, cluster load and problems.
 
-On the left hand side of the window you will see four further tabs.
-
-### Get Started Tab
-Provides an easy way to import the latest tweets from Twitters public timeline for a data set to experiment with.
+On the left hand side of the window you will see five further tabs.
```
Comment (bogensberger) [inline pages/docs/connect/admin_ui.html]: s/hand//

File: pages/docs/connect/admin_ui.html [hunk 2]
```diff
@@ -10,29 +10,24 @@
...
 
 - The name of the cluster, find out [how to change its value our configuration documentation](https://crate.io/docs/reference/configuration.html#cluster-name)
 - The version of Crate that the cluster is running.
-- The number of nodes in the cluster.
 - The health of the data in the cluster. Green indicates that all data is replicated and available, yellow that there are unreplicated records and red that data is unavailable.
 - Potential configuration problems and recommendations for the cluster.
 - Average cluster load over the past 1, 5 and 15 minutes.
-- Link to Crate documentation.
-- Latest news from Crate.
+- Settings Menu Which allows to update the language settings and provides a link to Crate documentation and the latest news from Crate.
```
Comment (bogensberger) [inline pages/docs/connect/admin_ui.html]: `Menu` and `Which` should start with small letters imho

Comment (nslater) [inline pages/docs/connect/admin_ui.html]: agree. though would replace "Which allows to" with "that allows you to"

also "to the CrateDB documentation"

"latest CrateDB news"

File: pages/docs/connect/admin_ui.html (hunk 1 above)
Comment (nslater) [inline pages/docs/connect/admin_ui.html]: "you will see" -> "there are"

also should there be an empty newline before this line?

File: pages/docs/connect/admin_ui.html [hunk 3]
```diff
@@ -43,6 +38,12 @@
 ### Cluster Tab
 Provides an overview of nodes in the cluster including details such as load, CPU usage, shards on the node and hostname.
 
+### Monitoring Tab
+Gives an overview of the average number and speed of CrateDB queries (`SELECT`, `INSERT`, `UPDATE` and `DELETE`). Please note that this feature is only available to the `Enterprise` customers.
```
Comment (nslater) [inline pages/docs/connect/admin_ui.html]: "only available to the `Enterprise` customers" -> "only available in the Enterprise Edition of CrateDB."

could maybe link "Enterprise Edition" to https://crate.io/pricing/

File: pages/docs/connect/admin_ui.html [hunk 4]
```diff
@@ -43,6 +38,12 @@
 ### Cluster Tab
 Provides an overview of nodes in the cluster including details such as load, CPU usage, shards on the node and hostname.
 
+### Monitoring Tab
+Gives an overview of the average number and speed of CrateDB queries (`SELECT`, `INSERT`, `UPDATE` and `DELETE`). Please note that this feature is only available to the `Enterprise` customers. 
+
+### Get Started Tab
+Provides an easy way to import the latest tweets from Twitters public timeline for a data set to experiment with.
```
Comment (nslater) [inline pages/docs/connect/admin_ui.html]: "Twitter's", or better: "the Twitter public timeline"

File: pages/docs/connect/admin_ui.html [hunk 5]
```diff
@@ -10,29 +10,23 @@
...
-- Latest news from Crate.
+- Settings menu that allows you to update the language settings and provides a link to the CrateDB documentation and the latest CrateDB news.
 
 The overview tab displays similar information to the bar but with more details on data availability, cluster load and problems.
-
-On the left hand side of the window you will see four further tabs.
-
-### Get Started Tab
-Provides an easy way to import the latest tweets from Twitters public timeline for a data set to experiment with.
+On the left side of the window here are five further tabs.
```
Comment (nslater) [inline pages/docs/connect/admin_ui.html]: "there are"

File: pages/docs/connect/admin_ui.html [hunk 6]
```diff
@@ -43,6 +37,12 @@
 ### Cluster Tab
 Provides an overview of nodes in the cluster including details such as load, CPU usage, shards on the node and hostname.
 
+### Monitoring Tab
+Gives an overview of the average number and speed of CrateDB queries (`SELECT`, `INSERT`, `UPDATE` and `DELETE`). Please note that this feature is only available in the [Enterprise Edition](https://crate.io/pricing/) of CrateDB. 
+
+### Get Started Tab
+Provides an easy way to import the latest tweets from Twitters public timeline for a data set to experiment with.
```
Comment (nslater) [inline pages/docs/connect/admin_ui.html]: "from the Twitter public timeline"

**Pull request merged**